### ISSUE TITLE
Nuevo libro Nivel 1, tema 1

### ISSUE THREAD

**Issue by cecilios** [lenmus/roadmap#14]
Os propongo abrir un _issue_ para lo que haya que comentar en cada tema del libro de nivel 1 realizado por Marcelino. Este primer _issue_ sería **exclusivamente** para comentar los textos del tema 1. Para otros temas abrimos el _issue_ correspondiente.

Tras revisar el tema 1, mi única sugerencia es sustituir la figura:

![t1-2](https://user-images.githubusercontent.com/5238679/81169243-cf10b100-8f98-11ea-950f-887b5bb7f089.png)

por esta otra:
![lineas-espacios](https://user-images.githubusercontent.com/5238679/81169316-f23b6080-8f98-11ea-88fc-a89fc8e5b2b1.png)

El motivo es la dificultad para las traducciones de la figura original, pues el tamaño de los textos "linea x" variará de un idioma a otro y al estar el texto antes de la flecha no puedo luego ajustar las flechas para que vayan inmediatamente tras el texto; además, me resulta muy complejo hacer que haya textos antes del pentagrama. Si los textos van tras las flechas, el tamaño del texto no influye en nada.

**Comment by u09644676** [comment]
Ningún problema. El gráfico que propones es perfectamente válido. (Habrá otros en LenMus que se podrán reutilizar en otros temas).

**Comment by cecilios** [comment]
En la mayoría de las figuras de este tema y del siguiente se dibujan las notas en los pentagramas utilizando cabezas de nota blanca, sin plica. En el tema 2 tambien se usan cabezas negras y blancas en vez de notas con plicas ¿Es necesario hacer esto así? ¿No podemos poner notas redondas en todos los sitios, hasta que se introduzcan las notas, las plicas y los corchetes en el tema 3?

**Comment by u09644676** [comment]
Sobre la cabezas de nota: En realidad es una cuestión 'menor', se podrían emplear solo 'redondas' hasta el tema 3, en lo cual a mi entender, no aporta ventajas alguna y sí cierta 'monotonía' visual. 
He empleado deliberadamente blancas y negras (unas veces sin plica otras con ellas) porque me parece muy útil que el alumno vaya viendo desde el principio figuras diferentes. Esto no supone ningún problema para lo que se está explicando (alturas, escalas, reconocimiento de notas) y por otra parte estimulará la curiosidad de algunos.
En cuanto a las blancas sin plica, si es problema dibujarlas, se pueden sustituir por 'redondas'.

**Comment by cecilios** [comment]
No, ningún problema para dibujarlas. Era solo que me extrañaba, sobre todo en el tema 2 en la escala de Fa donde todo son cabezas de negra menos una que es de blanca. Pensé que a lo mejor no era intencionado. Pero si lo quieres así no hay ningún problema.
